src/dht.cpp

```diff
@@ -1591,20 +1591,20 @@ Dht::Search::getUpdateTime(time_point now) const
 }
 
 bool
-Dht::Search::isAnnounced(Value::Id id, time_point now) const
+Dht::Search::isAnnounced(Value::Id id) const
 {
     if (nodes.empty())
         return false;
     unsigned i = 0;
     for (const auto& n : nodes) {
         if (n.isBad())
             continue;
-        if (not n.isAnnounced(id, now))
+        if (not n.isAnnounced(id))
             return false;
         if (++i == TARGET_NODES)
-            break;
+            return true;
     }
-    return i;
+    return false;
```

Comment:
Dang. I did cover this with my last version of the condition. I got confused removing it. Let me do that appropriately.